src/field/tracking/index.md

```diff
@@ -35,17 +35,21 @@ You can add new fields as needed, however, they should be set up with automatica
 
 ## Using position tracking in Mergin Maps mobile app
 
-::: warning
-Our mobile app was redesigned. We are in the process of updating this content to reflect these changes.
-:::
-
 To use tracking in the <MobileAppNameShort />, make sure that you enabled tracking in the QGIS project's settings. 
 
 1. Tap **More** and select the **Position tracking** option.
-   ![Mergin Maps mobile app Position tracking](./input-position-tracking.jpg "Mergin Maps mobile app Position tracking")
+   ![Mergin Maps mobile app Position tracking](./mobile-app-position-tracking.jpg "Mergin Maps mobile app Position tracking")
+
+2. Tap **Start tracking**.
+
+   ![Start tracking in Mergin Maps mobile app](./mobile-app-start-tracking.jpg "Start tracking in Mergin Maps mobile app")
+   
+3. The <MobileAppNameShort /> will continuously record your position in intervals defined in [**streaming mode settings**](../mobile-app-ui/#streaming-mode-and-recording-settings).
+
+   You can continue using the <MobileAppNameShort /> or other apps on your mobile device in the usual way. There will be an indicator that tracking is running in the status bar of your device.
```

Comment:
These intervals are actually not used for tracking, but for the streaming mode. Tracking interval is set in the QGIS plugin, see https://merginmaps.com/docs/field/tracking/#set-up-position-tracking-in-qgis-project